### PR TITLE
vtol fixes

### PR DESCRIPTION
**Pull request by sanderux**
Small fixes
- Force VTOL landing also during transitions (if land is engaged during front transition)
- Increase front transition duration limits

SITL tested.

### CODE REVIEW

**Comment by lamping7** [comment]
CI failed because it didn't land.
http://ci.px4.io:8080/blue/organizations/jenkins/Firmware/detail/PR-8958/2/pipeline/729

FCU never reported reaching the WP. This was the WP:
    frame: 3
    command: 21
    is_current: True
    autocontinue: True
    param1: 25.0
    param2: 0.0
    param3: 0.0
    param4: 0.0
    x_lat: 47.3992118835
    y_long: 8.54301234484
    z_alt: 12.0

This log (bad): https://logs.px4.io/plot_app?log=862a9e5e-89d3-402a-803f-baf7b729bea3
Known good log: https://logs.px4.io/plot_app?log=2b04575f-b7ae-43e7-ae63-29c613e3d913

**Comment by sanderux** [comment]
Runs a full mission fine in SITL here. I suspect a CI issue.
Log also showed a fully completed mission.
Lets see how it does now.

**Comment by lamping7** [comment]
I ran this locally, outside of rostest with QGC. The vehicle doesn't land after the transition. It just hovers at the landing WP.

I converted the mission to QGCs `.plan` file format so you can take a look. I had to rename it `.txt` for GitHub to accept it.
[vtol_old_2.txt](https://github.com/PX4/Firmware/files/1763433/vtol_old_2.txt)

**Comment by sanderux** [comment]
thanks, addressed

File: src/modules/navigator/navigator.h [hunk 1]
```diff
@@ -260,7 +260,7 @@ class Navigator : public control::SuperBlock
 	float		get_vtol_back_trans_deceleration() const { return _param_back_trans_dec_mss.get(); }
 	float		get_vtol_reverse_delay() const { return _param_reverse_delay.get(); }
 
-	bool		force_vtol() const { return _vstatus.is_vtol && !_vstatus.is_rotary_wing && _param_force_vtol.get(); }
+	bool		force_vtol() const { return _vstatus.is_vtol && (!_vstatus.is_rotary_wing || _vstatus.in_transition_to_fw) && _param_force_vtol.get(); }
```
Comment (dagar) [inline src/modules/navigator/navigator.h]: I'd say this is past being a one liner.

**Comment by sanderux** [comment]
@dagar addressed

**Pull request merged**